engine/api/application.go

```diff
@@ -247,6 +243,10 @@ func (api *API) getApplicationVCSInfosHandler() service.Handler {
 		}
 		resp.Remotes = remotes
 
+		if err := tx.Commit(); err != nil {
```

Comment:
Why did we need a DB transaction for this handler ?
It seems to be only just return data from the vcs

Reply:
`AuthorizedClient` takes a `gorpmapper.SqlExecutorWithTx` in his client. This client can be used to update the token / created date with `checkAccessToken`
All this code can be removed when the old vcs will be deleted.